CONTRIBUTING.md

```diff
@@ -74,6 +74,33 @@ eval "$(mise run beeai-platform:shell)"
 deactivate
 ```
 
+### Enabling or disabling auth flows for beeai_cli
+
+- Copy the apps/beeai-cli/.env.template as .env
+- To turn on oauth on platform startup set `BEEAI__OIDC_ENABLED=True` in the .env file.
+- Add an entry to /etc/hosts on your local system:
+```
+# Added by BEEAI-PLATFORM
+127.0.0.1        beeai-platform.api.testing
```

Comment:
The domain name is inconsistent.

- The TLD should be `.localhost` since everything `*.localhost` goes to localhost (at least on macOS and Ubuntu), so you could skip editing hosts file entirely
- Internally we use `beeai-server` for the API server, not `api`... but better yet it's the best candidate to serve the 2nd level domain anyway, so skip this segment
- Our domain is `beeai.dev` so let's be consistent and use `beeai.localhost`

So my suggestion for the domain is:
- change `beeai-platform.api.testing` -> `beeai.localhost` everywhere
- remove the instructions to edit `/etc/hosts` since that's not needed anymore